lib/seeing-is-believing.coffee

```diff
@@ -32,98 +58,62 @@ module.exports = SeeingIsBelieving =
   deactivate: ->
     @subscriptions.dispose()
 
-  configDefaults:
-    'ruby-command':  'ruby'
-    'add-to-env':
-      'LANG':        defaultLang
-      'ADD_TO_PATH': ''
-    'flags': [
-      '--alignment-strategy', 'chunk',
-      '--number-of-captures', '300',
-      '--line-length',        '1000',
-      '--timeout',            '12'
-    ]
-
-  invokeSib: (vars) ->
-    editor        = vars.editor
+  invokeSib: (additionalArgs) ->
+    editor        = atom.workspace.getActivePaneItem()
     crntBody      = editor.getText()
-    args          = ['-S', 'seeing_is_believing'].concat(vars.flags)
+    fileName      = editor.getPath()
+
+    mainCommand   = atom.config.get('seeing-is-believing.sibCommand')
+    args          = atom.config.get('seeing-is-believing.flags')
+    args.concat(additionalArgs) if additionalArgs?
```

Comment:
Concat here doesn't modify the array, it returns a new one, so the additional args won't work (thus `annotate-magic-comments` and `remove-annotations` don't work)